packages/jest-cli/src/reporters/IstanbulTestReporter.js

```diff
@@ -37,6 +40,44 @@ class IstanbulTestReporter extends DefaultTestReporter {
         reporter.addAll(config.coverageReporters);
         reporter.write(collector, true, () => {});
       } catch (e) {}
+      if (config.coverageThreshold) {
+        const rawCoverage = collector.getFinalCoverage();
+        const globalResults = istanbul.utils.summarizeCoverage(rawCoverage);
+        
+        function check(name, thresholds, actuals) {
+          return [
+            'statements',
+            'branches',
+            'lines',
+            'functions',
+          ].reduce((errors, key) => {
+            const actual = actuals[key].pct,
+                actualUncovered = actuals[key].total - actuals[key].covered,
+                threshold = thresholds[key];
```

Comment:
The code style is to use multiple variable declarations instead of just one. Would you mind making this change?

Is there an eslint rule for this? We should turn it on!